activemodel/test/cases/validations/length_validation_test.rb

```diff
@@ -385,10 +385,10 @@ def test_validates_length_of_using_both_minimum_and_maximum_should_not_allow_nil
     assert_predicate t, :invalid?
   end
 
-  def test_validates_length_of_using_minimum_0_should_not_allow_nil
+  def test_validates_length_of_using_minimum_0_should_allow_nil
     Topic.validates_length_of :title, minimum: 0
     t = Topic.new
-    assert_predicate t, :invalid?
+    assert_predicate t, :valid?
```

Comment:
You change the behavior on this case. This test should stay the same.

Reply:
revert my changes.